src/styles/saveLoadingPanelContent.scss

```diff
@@ -24,6 +24,7 @@
             .cmdrHeaderText {
                 display: flex;
                 flex-direction: column;
+                justify-content: center;
```

Comment:
I am sorry I was not precise enough, I was talking about `.saveText`, not `.cmdrHeaderText` 

![imageedit_2_8887016848](https://github.com/user-attachments/assets/1b2f3d51-3792-4fb5-9dac-521066ebe5cf)